bitgouel-domain/src/main/kotlin/domain/school/enums/SchoolConstant.kt

```diff
@@ -0,0 +1,24 @@
+package domain.school.enums
+
+/**
+ * 빛고을 직업교육 혁신지구 참여 학교 enum
+ */
+enum class SchoolConstant(
+    val schoolName: String
+) {
+
+    GWANGJU_TECHNICAL_HIGHSCHOOL("광주공업고등학교"),
+    KUMPA_TECHNICAL_HIGH_SCHOOL("금파공업고등학교"),
+    JEONNAM_TECHNICAL_HIGH_SCHOOL("전남공업고등학교"),
+    GWANGJU_GIRLS_COMMERCIAL_HIGH_SCHOOL("광주여자상업고등학교"),
+    JEONNAM_GIRLS_COMMERCIAL_HIGH_SCHOOL("전남여자상업고등학교"),
+    GWANGJU_NATURAL_SCIENCE_HIGH_SCHOOL("광주자연과학고등학교"),
+    GWANGJU_ELECTRONIC_TECHNICAL_HIGH_SCHOOL("광주전자공업고등학교"),
+    DONGIL_HIGH_SCHOOL_OF_FUTURE_SCIENCE_HIGH_SCHOOL("동일미래과학고등학교"),
+    SEOJIN_GIRLS_HIGH_SCHOOL("서진여자고등학교"),
+    SUNGUI_SCIENCE_TECHNOLOGY_HIGH_SCHOOL("숭의과학기술고등학교"),
+    SONGWON_GIRLS_COMMERCIAL_HIGH_SCHOOL("송원여자상업고등학교"),
+    GWANGJU_AUTOMATIC_EQUIPMENT_TECHNICAL_HIGH_SCHOOL("광주자동화설비마이스터고등학교"),
+    GWANGJU_SOFTWARE_MEISTER_HIGHSCHOOL("광주소프트웨어마이스터고등학교")
```

Comment:
```suggestion
    GWANGJU_TECHNICAL_HIGH_SCHOOL("광주공업고등학교"),
    KUMPA_TECHNICAL_HIGH_SCHOOL("금파공업고등학교"),
    JEONNAM_TECHNICAL_HIGH_SCHOOL("전남공업고등학교"),
    GWANGJU_GIRLS_COMMERCIAL_HIGH_SCHOOL("광주여자상업고등학교"),
    JEONNAM_GIRLS_COMMERCIAL_HIGH_SCHOOL("전남여자상업고등학교"),
    GWANGJU_NATURAL_SCIENCE_HIGH_SCHOOL("광주자연과학고등학교"),
    GWANGJU_ELECTRONIC_TECHNICAL_HIGH_SCHOOL("광주전자공업고등학교"),
    DONGIL_HIGH_SCHOOL_OF_FUTURE_SCIENCE_HIGH_SCHOOL("동일미래과학고등학교"),
    SEOJIN_GIRLS_HIGH_SCHOOL("서진여자고등학교"),
    SUNGUI_SCIENCE_TECHNOLOGY_HIGH_SCHOOL("숭의과학기술고등학교"),
    SONGWON_GIRLS_COMMERCIAL_HIGH_SCHOOL("송원여자상업고등학교"),
    GWANGJU_AUTOMATIC_EQUIPMENT_TECHNICAL_HIGH_SCHOOL("광주자동화설비마이스터고등학교"),
    GWANGJU_SOFTWARE_MEISTER_HIGH_SCHOOL("광주소프트웨어마이스터고등학교")
```

이렇게 HIGHSCHOOL -> HIGH_SCHOLL로 enum 명을 통일하자!는 말인 것 같슴다 애초에 이것도 확인해서 드렸어야 했는데... 🙄